src/ui/ui.rs

```diff
@@ -105,12 +105,13 @@ impl UI {
     pub fn init(
         app: &gtk::Application,
         rx: glib::Receiver<Message>,
+        size: Vec<&str>,
```

Comment:
Same as above. Also, maybe change it to be a tuple of `(i32, i32)` and then parse it on the other side (in `main.rs`).